views/components/ChartList.jsx

```diff
@@ -45,54 +51,117 @@ const ChartList = () => {
       console.error(error, '*****')
     }
   }
+  const handleShareChart = (chart) => setChartToShare(chart)
+  const shareWithUsers = async (chart_id, sharedWith) => {
+    try {
+      const {
+        data: { ok },
+      } = await shareChart(chart_id, sharedWith)
+
+      if (ok === 1) {
+        const { data } = await getCharts()
+        setChartList(data)
+        setChartToShare({})
+      }
+    } catch (error) {}
+  }
+  const handleChange = (event) =>
+    setChartToShare((chart) => ({ ...chart, sharedWith: event.target.value }))
+  const handleDelete = (userToDelete) => {
+    setChartToShare((chart) => ({
+      ...chart,
+      sharedWith: chart.sharedWith.filter(
+        (username) => username !== userToDelete
+      ),
+    }))
+  }
```

Comment:
do we need to call the API to remove the user when a user is deleted?

Reply:
good catch, I thought that a user would go to the submit button but they could click cancel and this update wouldn't reflect what they want.
https://github.com/AMP-SCZ/dpdash/pull/131/commits/e250b027af6997da38ebfdf1d05fb1a75107db72